packages/plugin-user/src/user.js

```diff
@@ -28,26 +28,42 @@ const User = SparkPlugin.extend({
   },
 
   /**
-   * Finishes the signup process for a user that signed up on a mobile device
+   * Activates a Spark user account and exchanges for user token.
    * @param {Object} options
-   * @param {string} options.encryptedQueryString
-   * @returns {Promise<Object>}
+   * @param {Object} options.verificationToken (required)
+   * @returns {Promise} Resolves with a userSession
    */
   activate(options) {
     options = options || {};
 
-    if (!options.encryptedQueryString) {
-      throw new Error(`\`options.encryptedQueryString\` is required`);
+    if (!options.verificationToken) {
+      throw new Error(`\`options.verificationToken\` is required`);
     }
+    let response;
 
     return this.request({
-      service: `atlas`,
-      resource: `users/email/activate`,
+      uri: this.config.activationUrl,
       method: `POST`,
       body: options,
-      requiresClientCredentials: true
+      auth: {
+        user: this.spark.config.credentials.oauth.client_id,
+        pass: this.spark.config.credentials.oauth.client_secret,
+        sendImmediately: true
+      },
+      withCredentials: true
     })
-      .then((res) => res.body);
+      .then((res) => {
+        response = res;
+        return this.getOauthCode();
+      })
+      .then((res) => {
+        const code = res.match(`<title>(.*)</title>`)[1];
```

Comment:
are you sure this works? i'm pretty sure when you pass a string to String#match(), it looks for a string match rather than converting it to a regex.

I think what you're looking for is `/<title>(.*?)</title>/`